scripts/ci_steps/prepare_protocol_and_unit_tests.sh

```diff
@@ -2,8 +2,12 @@
 
 set -e
 
+# Get rid of the generated services
+rm -rf Sources/Services/*
+
+# Regenerate the SDK manifest, with only runtime included
 cd AWSSDKSwiftCLI
-swift run AWSSDKSwiftCLI generate-package-manifest --exclude-aws-services ..
+swift run AWSSDKSwiftCLI generate-package-manifest ..
```

Comment:
Simply deleting the services & regenerating the manifest has the same effect as the now-deleted CLI option.